google-cloud-spanner/src/test/java/com/google/cloud/spanner/TransactionManagerImplTest.java

```diff
@@ -86,26 +83,42 @@ public void beginCalledTwiceFails() {
     when(session.newTransaction()).thenReturn(txn);
     assertThat(manager.begin()).isEqualTo(txn);
     assertThat(manager.getState()).isEqualTo(TransactionState.STARTED);
-    exception.expect(IllegalStateException.class);
-    manager.begin();
+    try {
+      manager.begin();
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done